app/registries/foreman/settings/general.rb

```diff
@@ -53,7 +53,7 @@
       full_name: N_('Show Experimental Labs'))
     setting('append_domain_name_for_hosts',
```

Comment:
Since you're not actually dropping the setting, maybe the commit message should be different? :)

Reply:
Yes, definitely.

Reply:
I'd suggest to rename the setting, rather than to redefine things. When things can have two names, it's always confusing and leads to brittle code. Something like `display_fqdn_for_hosts` is closer to what it actually does.

Reply:
> s. When things can have two names, it's always confusing and leads to brittle code. Something like `display_fqdn_for_hosts` is close

Agree, I didn't come up with a new name yet, but your pitch seems like a solid choice.